### PR TITLE
split mediaplayer code for lazy loading

### PR DESCRIPTION
**Pull request by ya7ya**
this introduces the concept of code-splitting explain https://reactjs.org/docs/code-splitting.html
This is useful to keep the initial needed code small enough.
as an exaxmple, I chose to separate the `paratii-mediaplayer` till it's needed to play a video. check out the article above and the code to get an idea of the change required.

__note__ this is also possible for webpacked libs , not just react ( `paratii-lib` ). checkout https://webpack.js.org/guides/code-splitting/ for more info.

### CODE REVIEW

**Comment by ya7ya** [comment]
Okay, i'm not gonna debug flow 😏 ..

**Comment by jellegerbrandy** [comment]
Ah, this is actually very straightfoward.
i think also @bent0b0x has used a pattern similar like this.

File: src/scripts/components/Play.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import React, { Component } from 'react'
 import { Events } from 'clappr'
 import styled from 'styled-components'
-import CreatePlayer from 'paratii-mediaplayer'
+// import CreatePlayer from 'paratii-mediaplayer'
```
Comment (bent0b0x) [inline src/scripts/components/Play.js]: Remove comment

**Comment by jellegerbrandy** [comment]
sorry about three flow errors - you forked from a broken branch. Should be fixed now

**Pull request merged**